### ISSUE TITLE
Help! Links must have discernible text

### ISSUE THREAD

**Issue by jpdevries** [modxcms/revolution#13485]
### Summary
The 2.x default theme utilizes icon fonts in an inaccessible manor:

```html
<a href="?a=help" title=""><i class="icon-question-circle icon icon-large"></i></a>
```

![](http://j4p.us/3S1U3F0i133H/Screen%20Shot%202017-07-05%20at%202.07.44%20PM.png)

It would be great to doing a sweep and fix this every where in the manager, but I think it is important for at least the Help! button to be accessible.

### Step to reproduce
Open the Manager

### Observed behavior
Can't access the help link using assistive technology

### Expected behavior
Should be able to access the help link using assistive technology

### Environment
MODX 2.x

### Solution

```html
<a href="?a=help" title="" aria-label="Help!"><i class="icon-question-circle icon icon-large"></i></a>
```

**Comment by OptimusCrime** [comment]
Isnt that already a MAB recommendation?

**Comment by alroniks** [comment]
Also actual for 3.x, the main menu should be enhanced to fit accessibility.
